controllers/controllers.js

```diff
@@ -0,0 +1,11 @@
+const {fetchTopics} = require('../models/topics')
+
+function getTopics(req, res ,next) {
+    return fetchTopics().then((topics) => {
+        res.status(200).send(topics)
```

Comment:
This is all good except for the format you are sending in, `topics` here is an array, because that is what the model is returning. The data you send from the controller should always be an object, with the array stored inside it on a semantic key, i.e.- `{ topics: topics }`